### ISSUE TITLE
How to combine autoregressive model and classification multi-task head in ranking model?

### ISSUE THREAD

**Issue by MoFHeka** [facebookresearch/generative-recommenders#30]
If I understand correctly, autoregressive model has a loss, and also multi-task dense layers followed autoregressive model has a weighted loss. How to combine them? 
And in ranking model, how to calculate the loss of autoregressive model? It's the same as LLM which is cross-entropy? Or a softmax?

**Comment by dst111dst** [comment]
same question here
And I'm confused about the loss function part, if I understand correctly, the loss part in the repo does not show things related to generation?
